src/components/LoadingIcon/LoadingIcon.tsx

```diff
@@ -97,9 +96,9 @@ const LoadingIcon = ({height = '10rem'}) => {
   );
 };
 
-LoadingIcon.propTypes = {
+interface LoadingIconProps {
   /** The height of the loading icon */
-  height: PropTypes.string
-};
+  height: string;
```

Comment:
This prop should be marked as optional since it's given a default value on line 41